src/Test/Plutip/Internal/BotPlutusInterface/Run.hs

```diff
@@ -77,15 +78,15 @@ runContract_ ::
   forall (w :: Type) (s :: Row Type) (e :: Type) (a :: Type) (m :: Type -> Type).
   (ToJSON w, Monoid w, MonadIO m) =>
   ClusterEnv ->
-  BpiWallet ->
+  WalletInfo ->
   Contract w s e a ->
   m ()
 runContract_ e w c = void $ runContract e w c
 
 runContract ::
   (ToJSON w, Monoid w, MonadIO m) =>
   ClusterEnv ->
-  BpiWallet ->
+  WalletInfo ->
```

Comment:
Changing type of this argument will break some docs.
Like [here](https://github.com/mlabs-haskell/plutip/blob/master/local-cluster/README.md?plain=1#L51-L57) and [here](https://github.com/mlabs-haskell/plutip/blob/master/docs/interactive-plutip.md?plain=1#L151-L154).

Not sure now how to arrange it better. Maybe `addSomeWallet` can return `WalletInfo`, so we sort of take out relation between "wallets that cluster uses" and BPI, as BPI is an implementation detail.